src/main/java/com/github/cloudyrock/mongock/Mongock.java

```diff
@@ -110,6 +114,19 @@ void execute() {
 
   }
 
+  Object o;
+
+  private Object getObject() {
+    if(o == null) {
+      synchronized (o) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this dangerous instance of double-checked locking. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2168)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') "o" is not "private final", and should not be used for synchronization. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2445)